backend/de.metas.business/src/main/java/de/metas/costrevaluation/ICostRevaluationDAO.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * #%L
+ * de.metas.business
+ * %%
+ * Copyright (C) 2022 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.costrevaluation;
+
+import de.metas.util.ISingletonService;
+import org.compiere.model.I_M_CostRevaluationLine;
+
+import java.util.List;
+
+public interface ICostRevaluationDAO  extends ISingletonService
+{
+	List<I_M_CostRevaluationLine> retrieveLinesByCostRevaluationId(int costRevaluationId);
+
+	I_M_CostRevaluationLine getCostRevaluationLineById (int costRevaluationLineId);
```

Comment:
pls introduce CostRevaluationLineId similar with BPartnerLocationId,
i.e. basically will contain the CostRevaluationId and repoId (of the line)

Reply:
just saw that u already introduced it. pls use it.